config/config_default-test.php

```diff
@@ -48,7 +48,7 @@
         'settings' => [
             'charset' => 'utf8',
```

Comment:
Actually, you could remove that line as `settings.charset` only triggers a [`SET NAMES $charset`](https://github.com/propelorm/Propel2/blob/b1f82623ea1b61c9b4246aa7cf9be4196a9652a0/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php#L147-L150) which is being overwritten by our custom queries.
All custom 'queries' are [executed subsequently](https://github.com/propelorm/Propel2/blob/b1f82623ea1b61c9b4246aa7cf9be4196a9652a0/src/Propel/Runtime/Adapter/Pdo/PdoAdapter.php#L123-L134).